src/verso-manual/VersoManual.lean

```diff
@@ -265,6 +265,8 @@ def traverse (logError : String → IO Unit) (text : Part Manual) (config : Conf
   let remoteContent ← updateRemotes false config.remoteConfigFile (if config.verbose then IO.println else fun _ => pure ())
   let mut state : Manual.TraverseState := {
     licenseInfo := .ofList config.licenseInfo,
+    extraCss := Std.HashSet.emptyWithCapacity |>.insertMany config.extraCss
+    extraJs := Std.HashSet.emptyWithCapacity |>.insertMany config.extraJs
```

Comment:
I'm guessing there's a prettier way to do this?

Reply:
I'd usually write the empty hash set with `{}`.

Reply:
Not off the top of my head, if there's no `ofList` defined.